### ISSUE TITLE
[Accessibility] Consider bypassing Enter key in terminal Accessible View

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#197411]
Type: <b>Feature Request</b>

CC @meganrogge 

In interactive shell, I encounter some questions that I need to hit enter key to choose/accept the default selection in terminal. Currently, only letter keys are bypassed to terminal input. Could you please consider Enter key as well?

VS Code version: Code - Insiders 1.85.0-insider (e55009850900a64eeb37b2228c79855ceff63914, 2023-11-03T15:11:12.179Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Remote OS version: Linux x64 4.18.0-477.27.1.el8_8.x86_64

**Comment by meganrogge** [comment]
Just merged a PR to include a whole bunch of other keys, and yes, `Enter` makes sense too

https://github.com/microsoft/vscode/blob/ca471a725a0855225c56b0d0fc3b778314fac69c/src/vs/workbench/contrib/accessibility/browser/accessibleView.ts#L781

**Comment by meganrogge** [comment]
Focus a terminal. `Alt+f2` to open the terminal accessible view. Hit `enter`. The terminal should be focused and the view should hide.
